reference/ingestion-tools/fleet/install-standalone-elastic-agent.md

```diff
@@ -116,22 +144,19 @@ To install and run {{agent}} standalone:
 
     :::{tab-item} DEB
     
-    ::::{tip}
-    You must run this command as the root user because some integrations require root privileges to collect sensitive data.
-    ::::
+    TIP: You must run this command as the root user because some integrations require root privileges to collect sensitive data.
 
     ```shell
     sudo systemctl enable elastic-agent <1>
     sudo systemctl start elastic-agent
     ```
+    
     1. The DEB package includes a service unit for Linux systems with systemd. On these systems, you can manage {{agent}} by using the usual systemd commands. If you don’t have systemd, run `sudo service elastic-agent start`.
     :::
     
     :::{tab-item} RPM
     
-    ::::{tip}
-    You must run this command as the root user because some integrations require root privileges to collect sensitive data.
-    ::::
+    TIP: You must run this command as the root user because some integrations require root privileges to collect sensitive data.
```

Comment:
```suggestion
```

As before, the TIP doesn't make sense to DEB and RPM as the agent is already installed, and systemctl commands are always handled by root.